client/app.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react';
+import { hot } from 'react-hot-loader/root';
+
+import { fetchPizza } from './api/index'
+
+const App = () => {
+  const [pizzaData, setPizzaData] = useState([])
+
+  useEffect(() => {
+  async function getData() {
+    const response = await fetchPizza()
+    const { data } = await response 
+    console.log(data)
+    setPizzaData([...data])
+  }
+
+  if(!pizzaData.length) getData()
```

Comment:
@AStashevskaya If I have no pizza in my db, requests page will stuck in fetching

![useEffect](https://user-images.githubusercontent.com/12561302/113598649-bcd24380-963d-11eb-98df-0b82a8ba96df.gif)